PythonTool/PathCreator.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+
+def PathCreator():
```

Comment:
All commands are relative to the PythonTool folder. I run MakeImagesTerminal.py each time from the aforementioned folder.